docs/openapi/components/schemas/common/MillTestReportCertificate.yml

```
@@ -190,75 +148,16 @@ example: |-
                 "type": [
                   "MechanicalProperty"
                 ],
-                "identifier": "ISO 1352",
-                "name": "Torque-controlled fatigue testing",
-                "description": "ISO 1352:2011 specifies the conditions for performing torsional, constant-amplitude, nominally elastic stress fatigue tests on metallic specimens without deliberately introducing stress concentrations. The tests are carried out at ambient temperature (ideally at between 10 °C and 35 °C) in air by applying a pure couple to the specimen about its longitudinal axis."
-              },
-              "measurement": {
-                "type": [
-                  "MeasuredValue"
-                ],
-                "value": "00.00",
-                "unitCode": "UNKNOWN"
-              }
-            },
-            {
-              "type": [
-                "Observation"
-              ],
-              "date": "2019-12-11T03:50:55Z",
-              "property": {
-                "type": [
-                  "ChemicalProperty"
-                ],
-                "name": "Zirconium"
-              },
-              "measurement": {
-                "type": [
-                  "MeasuredValue"
-                ],
-                "value": "60.811",
-                "unitCode": "P1"
-              }
-            },
-            {
-              "type": [
-                "Observation"
-              ],
-              "date": "2019-12-11T03:50:55Z",
-              "property": {
-                "type": [
-                  "MechanicalProperty"
-                ],
-                "identifier": "ISO 1352",
-                "name": "Torque-controlled fatigue testing",
-                "description": "ISO 1352:2011 specifies the conditions for performing torsional, constant-amplitude, nominally elastic stress fatigue tests on metallic specimens without deliberately introducing stress concentrations. The tests are carried out at ambient temperature (ideally at between 10 °C and 35 °C) in air by applying a pure couple to the specimen about its longitudinal axis."
+                "identifier": "ISO 180",
+                "name": "Izod Impact Strength Test",
+                "description": "ISO 180 defines the method used for pendulums to determine the impact resistance of a plastic specimen when supported in a cantilever configuration. Test results are used to evaluate the resilience of materials, typically plastics."
```

As the `materials` are `typically` (not always) `plastics`, the `specimens` shouldn't be described as only `plastic`.
```suggestion
                "description": "ISO 180 defines the method used for pendulums to determine the impact resistance of a specimen when supported in a cantilever configuration. Test results are used to evaluate the resilience of materials, typically plastics."
```

---

Updated in `MillTestReport.yml` and `MillTestReportCertificate.yml`